### DISCUSSION TITLE
How to manually Adjust Size and Scope of Trade.Mananger

### DISCUSSION THREAD

**Discussion by TLable** [MicroTrendsLtd/NinjaTrader8#67] [General]
Attempting to learn WPF elements, can someone link or direct to a basic layout set of rules that I will read then adapt the exsisting UserControlNinjaBuddy.xaml to allow spacing & features in a space to increase number of check boxes etc.

**Comment by MicroTrendsTom** [comment]
For advanced  programmers experienced in WPF it would be very easy to use the GUI project,  as discussed in the read me and  make any edits there visually then save them out or then take the changes and implement directly into the open source code version etc.

Generally teaching how to code and use WPF is out of scope here.   
Best way to learn is use google  and search for "learning WPF" or try stackoverflow.com
